### DISCUSSION TITLE
Why was the old branch removed from GitHub?

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#747] [General Discussion]
Hello,

I recently noticed that an older branch in our GitHub repository has been removed. This branch contained some important work and I am trying to understand why it was deleted.

Could someone please clarify:

1. What are the possible reasons an old branch might be removed?
2. Is there a way to track or retrieve details about the deletion (e.g., who removed it, when it was removed)?
3. Is it possible to recover the deleted branch?

Any help or insights would be appreciated. Thanks in advance!

Additional Information:
Repository: https://github.com/Sunbird-Ed/sb-cordova-plugin-utility.git#release-4.1.0

cc: @baljeetsingh @swayangjit @nidhikachhawa24

**Comment by HarishGangula** [comment]
@siddhishinde0723  This branch is merged to master 7 months back.
https://github.com/Sunbird-Ed/sb-cordova-plugin-utility/commit/8e441044f42d1971dbfceb124320d66a0904dfd4

**Reply by siddhishinde0723** [reply]
@HarishGangula I am sure there must be a way to got to specific release , as there would be different implementation on
different versions I wanted to know how do we go to past branch can you share, tags or commit it of respective branches .
